packages/app-project/src/screens/ProjectHomePage/components/ZooniverseTalk/ZooniverseTalk.js

```diff
@@ -0,0 +1,53 @@
+import { withResponsiveContext } from '@zooniverse/react-components'
+import counterpart from 'counterpart'
+import React from 'react'
+import { Chat } from 'grommet-icons'
+import { Box, Paragraph } from 'grommet'
+
+import en from './locales/en'
+import ContentBox from '../../../../shared/components/ContentBox'
+import WidgetHeading from '../.././../../shared/components/WidgetHeading'
+import JoinInButton from './components/JoinInButton'
+import RecentSubjects from './components/RecentSubjects'
+
+counterpart.registerTranslations('en', en)
+
+function ZooniverseTalk (props) {
+  const showSubjects = props.screenSize !== 'small'
+  const width = showSubjects ? '18%' : 'full'
+
+  return (
+    <ContentBox direction='row' gap='large'>
```

Comment:
The flex direction here needs to be responsive. Here how this looks now:

<img width="418" alt="Screen Shot 2019-07-31 at 10 45 04 AM" src="https://user-images.githubusercontent.com/5016731/62226883-84666680-b380-11e9-8b06-77797ed2c699.png">

Reply:
@beckyrother here's what this looks like with the flex direction rotated to column on the mobile size:

<img width="365" alt="Screen Shot 2019-07-31 at 11 27 16 AM" src="https://user-images.githubusercontent.com/5016731/62229842-35bbcb00-b386-11e9-9d6d-064f9bde5085.png">

Reply:
Perfect! ty

Reply:
The `screenSize` is injected into the props from the `withResponseContext` HOC, so that can be used to set the flex direction depending on the size.

Reply:
I've updated this and added a spec.

Reply:
I switched this to use a responsive grid, instead of flexbox. There's now a grid that has one column on small screens, two columns on larger displays.